### ISSUE TITLE
Link text for deprecation notice is not clear

### ISSUE THREAD

**Issue by dpersing** [Shopify/polaris-react#1801]
## Issue summary

Deprecation notices for app bridge components use ambiguous link text ("here") to link to resources that help consumers understand the decision-making process. For folks who use screen readers and/or have cognitive issues, the "here" text does not provide enough context to understand where the link will go. This type of link text doesn't meet [our own content guidelines for links](https://polaris.shopify.com/components/navigation/link), nor does it meet WCAG standards ([2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/WCAG21/#link-purpose-in-context), Level A).

## Expected behavior

Link text will clearly convey the purpose of the link.

## Actual behavior

The link in each instance of the deprecation statement has text that simply reads "here".

<img width="636" alt="Screen shot of an example deprecation notice" src="https://user-images.githubusercontent.com/1462085/60907398-4917c280-a22e-11e9-8bc0-0a4f1da4ef95.png">

## Steps to reproduce the problem

1. Navigate to any app bridge component `README.md` or style guide page (such as https://polaris.shopify.com/components/overlays/resource-picker#section-use-in-an-embedded-application-deprecated-)
1. View the link text.
